### PR TITLE
Fix oauth scope formatting

### PR DESCRIPTION
**Pull request by eatonphil**
Closes https://github.com/linode/manager/issues/2138

### CODE REVIEW

File: src/profile/integrations/components/PersonalAccessToken.js [hunk 1]
```diff
@@ -13,9 +13,17 @@ import { AuthScopeCell } from '~/components/tables/cells';
 import TimeDisplay from '~/components/TimeDisplay';
 import { OAUTH_SCOPES, OAUTH_SUBSCOPES } from '~/constants';
 
-import { EditPersonalAccessToken } from './';
+import { EditPersonalAccessToken } from './EditPersonalAccessToken';
 
 
+export function scopeFormat(scope) {
```
Comment (mparke) [inline src/profile/integrations/components/PersonalAccessToken.js]: cross component/view imports :( 

can it go in a shared place?

Comment (eatonphil) [inline src/profile/integrations/components/PersonalAccessToken.js]: Done

**Comment by abemassry** [comment]
invariant.js?4599:44 Uncaught Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in.

image

**Comment by eatonphil** [comment]
@abemassry I think that was broken on develop. But I just fixed it.

**Comment by rmcintosh** [comment]
🔥

**Pull request merged**